server/graphql/v2/object/PaymentMethod.js

```diff
@@ -62,13 +62,42 @@ export const PaymentMethod = new GraphQLObjectType({
             return pick(paymentMethod.data, ['email']);
           }
 
+          // Return HostCollectiveId for prepaid
+          if (paymentMethod.type === 'prepaid') {
+            return pick(paymentMethod.data, ['HostCollectiveId']);
+          }
+
           const data = paymentMethod.data;
           // white list fields to send back; removes fields like CustomerIdForHost
           const dataSubset = pick(data, ['fullName', 'expMonth', 'expYear', 'brand', 'country', 'last4']);
 
           return dataSubset;
         },
       },
+      limitedToCollectiveIds: {
+        type: new GraphQLList(GraphQLString),
+        resolve(paymentMethod) {
+          let encodedIds;
+          if (paymentMethod.limitedToCollectiveIds !== null) {
+            encodedIds = paymentMethod.limitedToCollectiveIds.map(id => {
+              return idEncode(id, 'account');
+            });
+          }
+          return encodedIds;
+        },
+      },
+      limitedToHostCollectiveIds: {
```

Comment:
To follow the direction of GraphQL API v2, this should return a List of `Host`s, not ids.

Frontend side GraphQL query would look something like:

```
paymentMethods {
  id
  limitedToHosts {
   id
   legacyId
   slug
  }
}
```